src/krati_Jain/Assignment6Numbers.java

```diff
@@ -0,0 +1,55 @@
+package krati_Jain;
+
+class Assignment6Numbers {
+	int number1;
+	int number2;
+	int number3;
+	int number4;
+	
+	void evenNumbers() {
+		//if ((num1 < 10) || (num1 > 15))
+			//System.out.println("Provided input number is not in the defined range i.e. 10 t0 30");
+		//else {
+		System.out.println("Even Numbers are:");
+		for(int number1 = 10; number1 <=15 ; number1++ ) { //Range -> 10 to 15
+			if((number1%2) == 0)
+				System.out.println(number1);
+			}
+		}
+	
+	void divisibleBy5() {
+		System.out.println("Divisible by 5, numbers are:");
+		for(int number2 = 10; number2 <=30; number2++ ) { //Range -> 10 to 15
```

Comment:
this is your local variable not instance variable, if you think your are using that number1 which is Instance variable then it's wrong. 
If you bit confused do let me know